app/src/main/java/org/mozilla/tv/firefox/telemetry/SentryIntegration.kt

```diff
@@ -28,8 +30,20 @@ import org.mozilla.tv.firefox.ext.serviceLocator
  * "Sentry DSN (amazonWebviewRelease): X_X"
  */
 object SentryIntegration {
-    fun init(context: Context) {
-        context.serviceLocator.settingsRepo.dataCollectionEnabled.observeForever { isEnabled ->
+
+    @VisibleForTesting(otherwise = NONE)
+    var isInit = false
+        private set
+
+    /**
+     * Initializes Sentry. This method should only be called once.
+     */
+    fun init(appContext: Context, serviceLocator: ServiceLocator) {
```

Comment:
If we only need the locator for the `SettingsRepo`, could we pass that instead of the `ServiceLocator`?